### PR TITLE
Add coq-of-ocaml version 2.3.0

### PR DESCRIPTION
**Pull request by clarus**
Add version 2.3.0 for coq-of-ocaml

### CODE REVIEW

File: packages/coq-of-ocaml/coq-of-ocaml.2.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+build: [
+  ["sh" "-c" "cd OCaml && ./configure.sh"] {coq:installed}
+  [make "-C" "OCaml" "-j%{jobs}%"] {coq:installed}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+install: [
+  [make "-C" "OCaml" "install"] {coq:installed}
+]
+depends: [
+  "dune" {>= "1.11"}
```
Comment (mseri) [inline packages/coq-of-ocaml/coq-of-ocaml.2.3.0/opam:17]:
```suggestion
  "dune" {>= "2.0"}
```
According to the dune-project file this now requires 2.0

Comment (mseri) [reply packages/coq-of-ocaml/coq-of-ocaml.2.3.0/opam]: @clarus can you please upstream this change?

Comment (clarus) [reply packages/coq-of-ocaml/coq-of-ocaml.2.3.0/opam]: Thanks! I did upstream the change in https://github.com/clarus/coq-of-ocaml/pull/152

**Comment by mseri** [comment]
Thanks

**Comment by clarus** [comment]
Thanks

**Pull request merged**